frontend/src/layout/panel-layout/ProjectTree/projectTreeLogic.tsx

```diff
@@ -455,6 +456,12 @@ export const projectTreeLogic = kea<projectTreeLogicType>([
                 clearScrollTarget: () => '',
             },
         ],
+        editingItemId: [
+            '',
+            {
+                setEditingItemId: (_, { id }) => id,
+            },
+        ],
```

Comment:
Nit: I recommend setting it to `null` instead of `''`. Write `editingItemId: [null as string | null, {...}]` to get the correct generated types.